dao/src/main/java/com/fit_goal/EventDtoFields.java

```diff
@@ -0,0 +1,11 @@
+package com.fit_goal;
+
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
```

Comment:
you can create a true utility class by making the constructor private `@NoArgsConstructor(access = AccessLevel.PRIVATE)
`

Reply:
refactored